metron-platform/metron-common/src/main/java/org/apache/metron/common/configuration/ConfigurationsUtils.java

```diff
@@ -247,19 +301,24 @@ public static void uploadConfigsToZookeeper(String globalConfigPath, String pars
   }
 
   public static void visitConfigs(CuratorFramework client, ConfigurationVisitor callback) throws Exception {
-    visitConfigs(client, callback, ConfigurationType.GLOBAL);
-    visitConfigs(client, callback, ConfigurationType.PARSER);
-    visitConfigs(client, callback, ConfigurationType.ENRICHMENT);
+    visitConfigs(client, callback, GLOBAL);
```

Comment:
Not a big deal, but maybe we could make visitConfigs take a vararg of ConfigurationType and then pass in `GLOBAL`, `PARSER`, `ENRICHMENT`, `PROFILER` in as args